packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "datacenter_backup_legacy_message_1": "Depuis le 25/02/2020, la gamme Veeam Backup a évolué.",
+  "datacenter_backup_legacy_message_2": "Vous pouvez choisir entre 3 nouvelles offres pour protéger vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
+  "datacenter_backup_legacy_message_3": "N’oubliez pas de mettre à jour votre solution !",
```

Comment:
```suggestion
  "datacenter_backup_legacy_message_3": "N’oubliez pas de mettre à jour votre solution !",
```
  
  non-breaking space

Reply:
Should we have the space or remove ? I dont get this..

Reply:
It should be a non-breaking space, not a normal space 🙂

Reply:
How to add nonbreaking space using the keyboard?

Reply:
Don't know if you can do this with your keyboard, for us it's `options + space`, apparently on Windows it's `ctrl + alt + space`

Reply:
Got it. Thanks. I think it works for us too.